src/getOldestInstance.ts

```diff
@@ -19,10 +19,10 @@ export async function handler(event) {
 let awsEc2 = new AWS.EC2();
 
 function getInstances(asgName: string): Promise<string[]> {
-    let autoScalingRequest = describeAsg(asgName);
+    const autoScalingRequest = describeAsg(asgName);
     return autoScalingRequest.then(
         function (data) {
-            let instances = data.AutoScalingGroups[0].Instances;
+            const instances = data.AutoScalingGroups[0].Instances;
```

Comment:
Slightly outside the scope of this PR, however do we need a guard clause here? 
- Will data always have a `AutoScalingGroups` key? 
- will the `AutoScalingGroups` never be an empty array?
- will the `Instances` always exist within that array?

Reply:
There are a couple of instances of this at the moment which need to be tidied up properly 👍